tox.ini

```diff
@@ -87,6 +87,7 @@ deps =
 allowlist_externals =
     bash
 commands =
+    python docs/src/gallery.py examples/ase
```

Comment:
Why do we need this line? Shouldn't the documentation including the gallery build by sphinx?

Reply:
Yes, correct. The gallery is now building automatically